test/functional/test_framework/test_node.py

```diff
@@ -91,6 +91,7 @@ def __init__(self, i, datadir, *, rpchost, timewait, bitcoind, bitcoin_cli, cove
             "-debugexclude=libevent",
             "-debugexclude=leveldb",
             "-uacomment=testnode%d" % i,
+            "-upnp=0",
```

Comment:
See the comment in line 84

`common args are set in the config file (see initialize_datadir)`. The args here are only to set the datadir and debug log options.

Reply:
Agree, looks like all tests use the generated bitcoin.conf.